src/qt5/abstract_list_model.cr

```diff
@@ -0,0 +1,13 @@
+abstract class Qt::AbstractListModel
+  def row_count : Int32
```

Comment:
Aren't `#row_count` and `#column_count` auto-generated by Bindgen?

Reply:
I thought so too, but these jump table entries dont seem to make it:

```
 bg_QAbstractItemModel_rowCount_const_QModelIndex_R: BindgenHelper.wrap_proc({% if forwarded.includes?("row_count") %} Proc(Binding::QModelIndex*, Int32).new{|parent| self.row_count(ModelIndex.new(unwrap: parent)) } {% else %} nil {% end %}),
      bg_QAbstractItemModel_columnCount_const_QModelIndex_R: BindgenHelper.wrap_proc({% if forwarded.includes?("column_count") %} Proc(Binding::QModelIndex*, Int32).new{|parent| self.column_count(ModelIndex.new(unwrap: parent)) } {% else %} nil {% end %}),
      ```

Reply:
Hmpf.  We can investigate this later I guess.